projectgenerator/libili2pg/iliimporter.py

```diff
@@ -92,15 +94,18 @@ def run(self):
             args += ["--dbpwd", self.configuration.password]
         args += ["--dbdatabase", self.configuration.database]
         args += ["--dbschema", self.configuration.schema or self.configuration.database]
-        args += ["--importTid"]
-        args += ["--nameByTopic"]
         args += ["--createEnumTabs"]
-        args += ["--createEnumColAsItfCode"]
         args += ["--createNumChecks"]
-        args += ["--smart1Inheritance"]
         args += ["--coalesceMultiSurface"]
         args += ["--createGeomIdx"]
         args += ["--createFk"]
+        if self.configuration.inheritance == 'smart1':
+            args += ["--smart1Inheritance"]
+        else:
+            args += ["--smart2Inheritance"]
```

Comment:
It shouldn't change the default. When reading from `QSettings` while initializing the dialog, the default value is `'smart1'`.